src/main/java/com/o19s/es/ltr/action/TransportAddFeatureToSetAction.java

```diff
@@ -181,7 +174,8 @@ private void fetchFeaturesFromStore() {
             srequest.source().query(bq);
             srequest.source().fetchSource(true);
             srequest.source().size(StoredFeatureSet.MAX_FEATURES);
```

Comment:
nit: perhaps add:
srequest.source().trackTotalHitsUpTo(StoredFeatureSet.MAX_FEATURES);
?
Certainly not useful but was just curious about this new optimization :)